guptahemant/insta-clone
# #91 Hashtags Implementation

### Objective: Implementing Hashtags Functionality on Instaclone

**Testing Steps:**
i. Create new post with image. Use hashtags starting with '#' symbol. Like, #nature, #tree, etc.
ii. Save the post. On post detailed page you will see highlighted hashtags links.
iii. Click on highlighted hashtag link. It should take you to yourdomain.com/explore/tags/<hashtag> page.
iv. The page will display currently accessed hashtag including how many posts are tagged with it in header section.

Desktop View:
![Screenshot from 2022-03-16 16-42-30](https://user-images.githubusercontent.com/2410377/158624982-7fbb1684-f5c1-475e-b675-b67781acef06.png)


Tablet View:
![Screenshot from 2022-03-16 20-46-16](https://user-images.githubusercontent.com/2410377/158624701-1df193c2-e075-45fe-b3d0-fb3d7659e650.png)

Mobile View:

![Screenshot from 2022-03-16 20-47-44](https://user-images.githubusercontent.com/2410377/158625152-fdbdb7e4-d885-49d2-83ad-dafea15ff318.png)
